protobuf-kafkaconnect-converter/src/test/java/com/amazonaws/services/schemaregistry/kafkaconnect/protobuf/ProtobufSchemaConverterTest.java

```diff
@@ -114,6 +117,7 @@ public void initializesConverter_Successfully() {
     @ParameterizedTest
     @MethodSource("getFromConnectTestCases")
     public void fromConnectData_convertsConnectDataToGSRSerializedProtobufData(
+        //TODO: Update this for all types, not just Primitive
```

Comment:
Why are we adding this comment? The test has already included all types right? Please make sure your commit is clean.